database/tables/instance_questions.pg

```diff
@@ -13,6 +13,8 @@ columns
     incremental_submission_points_array: double precision[]
     incremental_submission_score_array: double precision[]
     last_submission_score: double precision
+    manual_grading_conflict: boolean default false
+    manual_grading_user: bigint
```

Comment:
In terms of efficiency, I think your approach is better. I'm a bit uncertain of how to model this. 

I do not think we want to add this data to the `users` table. We could, perhaps, create a `users_grading` table to have a primary user key and an `instance_question` for manual grades. 

I suspect based on the feedback of the manual grading feature, we will eventually want to model this properly. In the meantime, I suppose it would be tentatively easy to drop a `users_manual_grading` table after doing a migration of that data.